### PR TITLE
Mod: remove some superfluous Boolean checks

### PR DESCRIPTION
**Pull request by donovaly**
Please have any Pythionist have a look and merge if OK.

### CODE REVIEW

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7213">pipeline status</a> for feature branch [PR_7213](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7213). Pipeline [590314416](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/590314416) was triggered at [0890be0](https://github.com/FreeCAD/FreeCAD/pull/7213/commits/0890be09a3aa5355a3cc7d246d3356e6ba71c0f3). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

**Comment by berndhahnebach** [comment]
<a href="https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7213">pipeline status</a> for feature branch [PR_7213](https://gitlab.com/freecad/FreeCAD-CI/-/commits/PR_7213). Pipeline [590322400](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines/590322400) was triggered at [cd0e442](https://github.com/FreeCAD/FreeCAD/pull/7213/commits/cd0e4422ee3753b0e74204bf01c7f34bafa61ee1). All CI [branches](https://gitlab.com/freecad/FreeCAD-CI/-/branches/all) and [pipelines](https://gitlab.com/freecad/FreeCAD-CI/-/pipelines?scope=branches).

File: src/Mod/Arch/ArchSite.py [hunk 1]
```diff
@@ -520,7 +520,7 @@ def Activated(self):
             if (Draft.getType(obj) == "Building") or (hasattr(obj,"IfcType") and obj.IfcType == "Building"):
                 siteobj.append(obj)
             else :
-                if link == True :
+                if link :
```
Comment (benj5378) [inline src/Mod/Arch/ArchSite.py:523]:
```suggestion
                if link:
```

File: src/Mod/Draft/draftfunctions/upgrade.py [hunk 2]
```diff
@@ -507,7 +507,7 @@ def makeWires(objectslist):
             elif len(objects) > 1 and len(edges) > 1:
                 result = makeWires(objects)
                 if result:
-                    if result == True:
+                    if result:
```
Comment (benj5378) [inline src/Mod/Draft/draftfunctions/upgrade.py:510]:
```suggestion
                if makeWires(objects):
                    _msg(translate("draft","Found several wires or edges: wiring them"))
                else:
                    _msg(translate("draft","Found several non-treatable objects: creating 
```
Bonus points for refactoring this

**Review by benj5378 — APPROVED** [review]

File: src/Mod/Arch/ArchBuilding.py [hunk 3]
```diff
@@ -231,7 +231,7 @@ def Activated(self):
             if not Draft.getType(obj) in ["Site", "Building"] :
                 buildingobj.append(obj)
             else :
-                if link == True :
+                if link :
```
[the remark this answers is not in the document]
Comment (donovaly) [reply src/Mod/Arch/ArchBuilding.py:234]: But the other ':' in the file all have a preceding space. When I change this one, I will have to change all others -> can be done in a separate PR.

File: src/Mod/Draft/draftfunctions/upgrade.py (hunk 2 above)
Comment (donovaly) [reply src/Mod/Draft/draftfunctions/upgrade.py:510]: done

**Comment by donovaly** [comment]
Thanks for reviewing! Except of one suggestions it is in.

Please open a new PR fir further code changes like removal of unnecessary spaces.

**Pull request merged**